src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -78,8 +74,6 @@ public DissolutionCreateResponse submitDissolutionRequest(
             @Valid @RequestBody final DissolutionCreateRequest body,
             HttpServletRequest request) {
 
-        LOGGER.info("[POST] Submitting dissolution request for company number {}", companyNumber);
```

Comment:
Now logging request start and end, so no longer need to manually add these logs.